superset/views/geocoding.py

```diff
@@ -0,0 +1,327 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=C,R,W
+
+import simplejson as json
+from flask import request, Response
+from flask_appbuilder import expose
+from flask_appbuilder.security.decorators import has_access, has_access_api
+from flask_babel import gettext as __
+from sqlalchemy import Column, Float, text
+from sqlalchemy.engine import Connection, reflection
+
+import superset.models.core as models
+from superset import appbuilder, conf, db
+from superset.connectors.sqla.models import SqlaTable
+from superset.exceptions import (
+    SqlAddColumnException,
+    SqlSelectException,
+    SqlUpdateException,
+)
+from superset.utils.geocoding_utils import GeoCoder
+
+from .base import api, BaseSupersetView, json_error_response, json_success
+
+
+class Geocoder(BaseSupersetView):
```

Comment:
Almost same class-name as the utils class GeoCoder, should we change one of these or does it make sense to have them very similarly named?